helm-charts/helm2/strimzi-kafka-operator/templates/_helpers.tpl

```diff
@@ -31,19 +31,6 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
-{{/*
```

Comment:
This appears unused in both `_helpers.tpl` so I went ahead and cleaned it up. This is split into a separate commit and I can remove it if we don't want this change.